DependencyInjection/Configuration.php

```diff
@@ -40,10 +40,24 @@ public function getConfigTreeBuilder()
                     ->requiresAtLeastOneElement()
                     ->useAttributeAsKey('name')
                     ->prototype('array')
-                        ->children()
+						->children()
                             ->scalarNode('title')->isRequired()->end()
                             ->scalarNode('description')->isRequired()->end()
-                            ->scalarNode('link')->isRequired()->end()
+							->arrayNode('link')
+								->isRequired()
+								->beforeNormalization()
+									->ifString()
+									->then(function ($v) { return [ 'direct' => $v]; })
```

Comment:
Can you please remove the short array annotation with the following please (in order to be compliant with PHP < 4.0.0):

```php
->then(function ($v) { return array('direct' => $v); })
```